NtupleProducer/python/scripts/objMultiplicityPlot.py

```diff
@@ -3,79 +3,93 @@
 import ROOT
 ROOT.PyConfig.IgnoreCommandLineOptions = True
 ROOT.gROOT.SetBatch(True)
-ROOT.gStyle.SetOptStat(False)
+ROOT.gStyle.SetOptStat(0)
+ROOT.gStyle.SetOptTitle(0)
 ROOT.gStyle.SetErrorX(0.5)
 ROOT.gErrorIgnoreLevel = ROOT.kWarning
-
 from math import *
-
 from optparse import OptionParser
+import sys
+sys.path.insert(0,'./')
+from display.examplePlot import *
```

Comment:
??
why not `from FastPuppi.NtupleProducer.scripts.display.examplePlot import *` ?